src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/CompositionManager.cs

```diff
@@ -499,6 +536,12 @@ private void Update()
                         CurrentCompositeFrame = 0;
                         timeSynchronizer.Reset();
                         poseCache.Reset();
+
+                        if (UnityCompositorInterface.IsCameraCalibrationInformationAvailable())
+                        {
+                            stationaryCameraBroadcaster.SetActive(true);
+                            HolographicCameraObserver.Instance.ConnectTo("127.0.0.1");
+                        }

```

Comment:
could this be achieved without creating a connection to localhost/through establishing some sort of artificial connection?